analyses/rna-seq-expression-summary-stats/README.md

```diff
@@ -0,0 +1,181 @@
+## Calculate TPM summary statistics within each cancer group and cohort
+
+**Module authors:** Yuanchao Zhang ([@logstar](https://github.com/logstar))
+
+### Purpose
+
+Within each cancer group and cohort, calculate TPM means, standard deviations, z-scores, and ranks.
+
+### Methods
+
+Select independent RNA-seq samples using `independent-specimens.rnaseq.primary.tsv` in the results of the `independent-samples` analysis module.
```

Comment:
```suggestion
Select independent RNA-seq samples using `independent-specimens.rnaseq.primary.tsv` in the results of the `independent-samples` analysis module.
```

Note to update this after https://github.com/PediatricOpenTargets/ticket-tracker/issues/89

Reply:
Thanks. Will update in the next patch.